scripts/lib/CIME/provenance.py

```diff
@@ -71,9 +86,18 @@ def _save_build_provenance_e3sm(case, lid):
     env_module = case.get_env("mach_specific")
     env_module.save_all_env_info(env_prov)
 
+    # Save build times
+    build_times = os.path.join(exeroot, "build_times.{}.txt".format(lid))
+    if os.path.exists(build_times):
+        os.remove(build_times)
+    globstr = "{}/*bldlog*{}.gz".format(exeroot, lid)
+    matches = glob.glob(globstr)
+    if matches:
+        _extract_times(matches, build_times)
```

Comment:
I presume this means the overall component build times are written at the end of this file, right?

Reply:
@sarats , remember that, with CMake, the entire build is done at once, not component-by-component (unless --separate-builds is provided to case.build). So we can't code our provenance to expect per-component build times.

Reply:
Oh right, I was looking at Az's output that probably used separate builds.